### PR TITLE
Add the concordance page.

### PR DESCRIPTION
**Pull request by sarum90**
This PR adds another route /dict/concordance/<morpheme>/<definition> which will render the dictionary on the left, and the concordance of a specific morpheme  / definition on the right.

It also modifies the morphemes in the dictionary to be links to the concordance page.

### CODE REVIEW

File: moroScript.jsx [hunk 1]
```diff
@@ -255,24 +279,50 @@
...
         }
       });
 
+      // Dictionary view with concordance.
+      var ConcordanceView = React.createClass({
+        render: function() {
+          var text = "Definition for: " + this.props.params.morpheme + 
+            " is " + this.props.params.definition;
+          return <div className="ui segment">
+                {text}
```
Comment (sarum90) [inline moroScript.jsx]: After {text} and before </div> would be the location to render the sentences that have the specified morpheme and definition.

**Comment by sarum90** [comment]
@sande570 - This PR is what I showed you earlier, with the dictionary only rendering on the left half of the pane, and the right half of the pane having a place for the concordance to be rendered.

**Comment by psejenks** [comment]
Thanks so much for your help with this (and all other aspects of the project), Marcus!

**Comment by sarum90** [comment]
Not ready for merge yet, will add rendering sentences sometime this weekend.

**Comment by sarum90** [comment]
Just found this site: https://rawgit.com/

We can use it to view branches before they are merged to the gh-pages branch:

https://rawgit.com/sande570/Moro-database/preparing_for_concordance/index.html#/dict

**Comment by sarum90** [comment]
Merging since concordance is done now.

**Pull request merged**